cmd/zoekt-webserver/main.go

```diff
@@ -124,8 +131,34 @@ func main() {
 	templateDir := flag.String("template_dir", "", "set directory from which to load custom .html.tpl template files")
 	dumpTemplates := flag.Bool("dump_templates", false, "dump templates into --template_dir and exit.")
 	version := flag.Bool("version", false, "Print version number")
+	useJaeger := flag.Bool("jaeger", false, "Collect and send traces to Jaeger. Jaeger configuration set via environment variables. Only traces for RPC requests that explicitly opt into traces will be collected.")
```

Comment:
This file contains command-line flags to enable Jaeger.

Reply:
I think it would be nice to enable/disable Jaeger without shipping a new binary, and I think the flag adds a minor amount of complexity (removing it would eliminate ~2 lines of code).